src/html.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I think the helmet plugin along with react-helmet will be the way to go for this. You can add it to the layouts and avoid having this here. This file should only bee needed if you need to rearrange where things are going.

Reply:
Was able to get this working using the approach you suggested. Much cleaner now, thanks!!